packages/games/emulators/raze/sources/start_raze.sh

```diff
@@ -24,7 +24,7 @@ mkdir -p ${SAVE_DIR}
 params=" -config ${CONFIG} -savedir ${SAVE_DIR}"
 params+=" +gl_es 1 +vid_preferbackend 3 +cl_capfps 0 +cl_nomeleeblur 1 +vid_fps 1"
 
-EXT=${1#*.}
+EXT=${1##*.}
```

Comment:
Unrelated to this specific PR -- but same issue for build eng games so may as well fix. I did a search for any lines that matched this-- this was all of them.